add missing requirements to the README (#246)
stripe/stripe-react-native: README.md

#### Android

- Android 5.0 (API level 21) and above
- Android gradle plugin 4.x and above

#### iOS

Compatible with apps targeting iOS 11 or above.

> sdk uses typescript features available from `>= 7.9.0` babel version,
> you can either upgrade babel or use `plugin-transform-typescript` plugin to handle this.

Reviewer: ```suggestion
The SDK uses TypeScript features available in Babel version `7.9.0` and above.
Alternatively use the `plugin-transform-typescript` plugin in your project.
```
